### PR TITLE
Bugfix/hotspot details

### PR DESCRIPTION
**Pull request by matthewcarlreetz**

### CODE REVIEW

File: src/features/hotspots/details/HotspotDetails.tsx [hunk 1]
```diff
@@ -426,22 +429,32 @@ const HotspotDetails = ({
     return makers?.find((m) => m.address === hotspot?.payer)?.name
   }, [hotspot?.payer, makers])
 
+  const bottomSheetBackground = useCallback(
+    () => <Box flex={1} width="100%" backgroundColor="yellow" />,
```
Comment (tyler-whitman) [inline src/features/hotspots/details/HotspotDetails.tsx:433]: What is this for?

Comment (matthewcarlreetz) [reply src/features/hotspots/details/HotspotDetails.tsx:433]: If you...
1. expand the hotspot bottom sheet 
2. scroll down
3. pull the handle down to collapse the bottom sheet
You can end up in the collapsed state with the scrollview still scrolled to the bottom.

So, I moved the handle to the main container and added a background view so it wasn't transparent.
Simulator Screen Shot - iPhone 13 - 2021-11-29 at 11 15 04

Comment (tyler-whitman) [reply src/features/hotspots/details/HotspotDetails.tsx:433]: Ah ok, why yellow?

**Review by tyler-whitman — APPROVED** [review]

File: src/features/hotspots/details/HotspotDetails.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/hotspots/details/HotspotDetails.tsx]: oh, I just had that in there for testing. Will remove.

**Pull request merged**